services/organization_service/README.md

```
@@ -0,0 +1,52 @@
+   ```bash
+   cd services/organization_service
+   ```
+3. Install the dependencies:
+   ```bash
+   npm install
+   ```
+
+## Usage
+
+1. Create a `.env` file in the root of the `organization_service` directory and add the following environment variables:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)